docs/platforms/python/integrations/tornado/index.mdx

```diff
@@ -84,4 +84,11 @@ It takes a couple of moments for the data to appear in [sentry.io](https://sentr
 ## Supported Versions
 
 - Tornado: 5+
-- Python: 2.7+
+- Python: 3.7+
```

Comment:
just double checking this should indeed be python 3.7+ since everything else is 3.6